bats/katello-bats

```diff
@@ -34,8 +34,8 @@ for arg do
       swapfile)
         swapfile=true
         ;;
-      capsule)
-        katello_capsule=true
+      proxy)
+        katello_proxy=true
```

Comment:
Oh I thought we could keep it katello_proxy for now, it's going to go anyway with #334.